code/datums/ai/_ai_controller.dm

```diff
@@ -130,6 +132,54 @@ multiple modular subtrees with behaviors
 	RegisterSignal(pawn, COMSIG_MOB_STATCHANGE, PROC_REF(on_stat_changed))
 	RegisterSignal(pawn, COMSIG_MOB_LOGIN, PROC_REF(on_sentience_gained))
 	RegisterSignal(pawn, COMSIG_QDELETING, PROC_REF(on_pawn_qdeleted))
+	RegisterSignal(pawn, COMSIG_GRID_UPDATED, PROC_REF(update_grid))
+
+	our_cells = new(interesting_dist, interesting_dist)
```

Comment:
also, cell tracker datums have this system of a "grace window". Basically we hold onto cells that are outside our bounds but inside a second, larger set of bounds. This is in the interest of avoiding a bunch of churn (cause I made it for ventcrawling and ventcrawling is speedy as hell).

If you want member_cells to be the cells in bound and ONLY the cells in bound, you need to give this a ratio of 1 (instead of the default of 2).
This way we avoid staying active because we used to be somewhere, and there's still a dude there.